.github/workflows/linux.yml

```diff
@@ -62,6 +59,17 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
 
+    - name: Install Boost
+      run: |
+        mkdir -p boost
+        cd boost
+        wget -O boost_1_87_0.tar.bz2 https://boostorg.jfrog.io/artifactory/main/release/1.87.0/source/boost_1_87_0.tar.bz2
+        tar --bzip2 -xf boost_1_87_0.tar.bz2
+        cd boost_1_87_0
+        export PKG_CONFIG_PATH=$PWD/boost
+        ./bootstrap.sh --prefix="boost" --show-libraries --with-libraries='math'
+        ./b2 install --prefix="boost"
```

Comment:
Meson doesn't find boost here - perhaps we just need to set `PKG_CONFIG_PATH` and/or `--prefix=` to different values?

Reply:
Can't we get this 1.87.0 conda-forge? This seems like a bit of a maintenance hassle. We have an existing CI job with conda-forge, so just adding it there seems like the most pain-free option.

Reply:
ah, that didn't cross my mind!

Reply:
https://github.com/conda-forge/boost-feedstock/pull/221